src/eu/packages/eu-component-icon/docs/icon.md

```diff
@@ -0,0 +1,40 @@
+# ECL Twig - EC Icon component
```

Comment:
there should be no reference to EC in this file (including npm package below, and so on)